commands/operator-sdk/cmd/up/local.go

```diff
@@ -65,8 +66,12 @@ kubernetes cluster using a kubeconfig file.
 	upLocalCmd.Flags().StringVar(&operatorFlags, "operator-flags", "", "The flags that the operator needs. Example: \"--flag1 value1 --flag2=value2\"")
 	upLocalCmd.Flags().StringVar(&namespace, "namespace", "default", "The namespace where the operator watches for changes.")
 	upLocalCmd.Flags().StringVar(&ldFlags, "go-ldflags", "", "Set Go linker options")
-	if projutil.GetOperatorType() == projutil.OperatorTypeAnsible {
+	switch projutil.GetOperatorType() {
+	case projutil.OperatorTypeAnsible:
 		ansibleOperatorFlags = flags.AddTo(upLocalCmd.Flags(), "(ansible operator)")
+		zapFactory = zap.FactoryForFlags(upLocalCmd.Flags())
+	case projutil.OperatorTypeHelm:
+		zapFactory = zap.FactoryForFlags(upLocalCmd.Flags())
```

Comment:
This is only for helm or ansible?

Reply:
The creator of a golang operator would have to implement the code to do flags themselves and would need to use the `--extra-flags` to pass it to their code I believe

Reply:
The code for the zap flags is included in the scaffold in this PR, so that piece would be done already unless users change the flag initialization in our scaffold. But I concur with Shawn, `--extra-flags` is required because of the way `up local` works for go operators.